config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -143,7 +144,8 @@ module MOM_cap_mod
 integer              :: scalar_field_count = 0
 integer              :: scalar_field_idx_grid_nx = 0
 integer              :: scalar_field_idx_grid_ny = 0
-character(len=*),parameter :: u_FILE_u = &
+integer              :: nthrds  !< number of openmp threads per task
+character(len=*),parameter :: u_file_u = &
```

Comment:
I think, for consistency, we should change `u_file_u` back to `u_FILE_u`. (Not sure why it's changed in the first place.)